internal/commands/create.go

```diff
@@ -46,6 +41,37 @@ func newRegoFile(filePath string, contents string) (regoFile, error) {
 	return regoFile, nil
 }
 
+func newPolicyFile(filePath string, contents string) (regoFile, error) {
```

Comment:
thought: At a glance, I think it should be possible to get rid of both `newPolicyFile` and `newLibraryFile`.

Right now `newPolicyFile` will return an empty `regoFile` if the rego file does not have a violation rule. A lack of a violation rule means it's not a policyfile. But then we have a `loadPolicyFiles` that checks if the regoFile is blank. So we have a scenario where two methods have to know how to decern whether or not a rego file is a policy file or not.

I would attempt to leverage _only_ the loadLibraries and loadPolicies methods. This makes it such that only a single method needs to know what makes a rego file a library, and a single method that needs to know what a policy file is.